app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,27 @@
+<%= stylesheet_link_tag "recipes", "data-turbo-track": "reload" %>
+
+<h2>Recipes</h2>
```

Comment:
[OPTIONAL] Kindly ensure that you have one `h1` tag in each page. It is a good practice and helps SEO